server/build.go

```diff
@@ -275,9 +268,6 @@ func PostApproval(c *gin.Context) {
 		c.String(500, "cannot decline a build with status %s", build.Status)
 		return
 	}
-	build.Status = model.StatusPending
```

Comment:
:heavy_check_mark:

Reply:
Was a bit strange that the code update some status, than did some other things not related to the status change and than updated the status in the store :)